src/components/NamePicker.tsx

```diff
@@ -1,10 +1,76 @@
-import React from 'react';
-import TextField from '@material-ui/core/TextField';
+import React, { useState, MouseEvent, ChangeEvent } from 'react';
 import Grid from '@material-ui/core/Grid';
+import Popover from '@material-ui/core/Popover';
+import TextField from '@material-ui/core/TextField';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
 import { makeStyles } from '@material-ui/core/styles';
 
 export default function NamePicker(): JSX.Element {
   const classes = useStyles();
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const [anchorEl2, setAnchorEl2] = useState<null | HTMLElement>(null);
+  const [disabled1, setDisabled1] = useState(true);
```

Comment:
Can you use more descriptive names than '1' and '2'? Maybe 'first' and 'last'? You don't want the names to get too long and unwieldy, but I think 1 and 2 are going to be confusing and error-prone.

I wonder which is best: `lastDisabled` or `disabledLast` or `lastNameDisabled`? None of them are great, but I think any of them would be better than `disabled1`.

Reply:
Unfortunately my internet has been unstable today. 

But I have changed the name to firstDisabled, setFirstDisabled, lastDisabled , and setLastDisabled. Thank you for pointing this out, I need better habits!